docs/docs/develop/policies.md

```diff
@@ -0,0 +1,118 @@
+    ```yaml
+    policy:
+      has_access: "'{{ .user.admin }}' == true"
+      filter: "domain = '{{ .user.domain }}'"
+     ```
+   > **_Note:_**  `filter` condition needs to be a valid SQL `WHERE` clause. It will be injected in the queries after resolving the user attributes templating.
+3. Along with the above policy, you want to exclude certain dimension for user not having `test.com` domain. Assuming you have a `ssn` dimension in your model. You can define the following policy in the `dashboard_name.yaml` file under `policy` section.
+    ```yaml
+    policy:
+      has_access: "'{{ .user.admin }}' == true"
+      filter: "domain = '{{ .user.domain }}'"
+      exclude:
+        - name: ssn
+          if: "{{ .user.domain }} != 'test.com'"
+    ```
+   As per above ssn dimension will be excluded but all other dimensions will be included.
+4. Here's one example for column inclusion.
+
+    ```yaml
+    policy:
+      has_access: "true"
+      filter: "domain = '{{ .user.domain }}'"
+      include:
+        - name: ssn
+          if: "'{{ .user.admin }}' == true"
+    ```
+   > **_Note:_** If include is defined all other dimensions will be excluded. Only one of the
+     `include`/`exclude` can be defined at a time.
+
+> **_Note:_** `has_access` and `if` conditions on `include`/`exclude` fields needs to be valid expressions that evaluates to true/false.
+
+## Advanced Examples
+Policies are resolved in two phases - 
+1. The templating engine replaces placeholder like this `{{ }}` with actual values
+2. For `has_access` and `if` conditions, the resultant expression is evaluated to true/false, whereas for `filter` condition the resultant expression is injected in the query.
+
+> **_Note:_** We use sprig templating engine for templating and govaluate for expression evaluation. So any valid operator/expression they support will work here. You can refer to their documentation for more details.
+[Here](http://masterminds.github.io/sprig/) are the supported sprig functions and [here](https://github.com/Knetic/govaluate/blob/master/MANUAL.md#operators) are the supported expression evaluation operators. 
+
+Now lets look at some advanced examples.
+
+1. Let's say you want to restrict access to a dashboard only to admin users or users belonging to `test.com` domain.
+    ```yaml 
+    policy:
+      has_access: "'{{ .user.admin }}' == true || '{{ .user.domain }}' == 'test.com'"
+    ```
+2. Let's say additionally we want to filter queries based on user's groups and there exist a `group` dimension in the model.
+    ```yaml 
+    policy:
+      has_access: "'{{ .user.admin }}' == true || '{{ .user.domain }}' == 'test.com'"
+      filter: "groups IN ('{{ .user.groups | join "', '" }}')"
+    ```
+
+### Custom attributes
+Sometimes you may want to use custom attributes for defining policies other than existing user attributes. For example, you may have a `department` dimension and want to filter the dashboard view as per the users department.
```

Comment:
Can we make the example `tenant_id` instead? I think that would show a more powerful use case (partner filtering)